rizza/helpers/config.py

```diff
@@ -74,6 +84,33 @@ def load_config(self, cfg_file=None):
         """Attempt to load in config files"""
         infile = Path(cfg_file or self.cfg_file).resolve()
         logger.info(f"Loading config from {infile.absolute()}")
+
+        # Try to use nanoconf first if we have a .nconf file
```

Comment:
Let's get rid of the compatibility with non-nanoconf configs and just use nanoconf.